From f9be20b5bf7a3adbb3af36d60c4afa0b37443b76 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 07 Mar 2024 19:06:20 +0200
Subject: [PATCH] [SW-176257] gaudi3: change i2c slave addr to 0x67

Per customer's request, change Gaudi's I2C address to 0x67 instead
of the previous 0x6a.

Change-Id: I97ee4c1bf2f53440a6bf8c133a1ba3b0765b244e
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.h b/arch/gaudi3/arc/common.h
index 066c324..cb80322 100644
--- a/arch/gaudi3/arc/common.h
+++ b/arch/gaudi3/arc/common.h
@@ -75,7 +75,7 @@
 #define PPBOOT_DCCM_BASE		0x80006300
 
 #define FUSE_WORDS_PER_BANK		0x80
-#define I2C_SLAVE_ADDRESS		0x6A
+#define I2C_SLAVE_ADDRESS		0x67
 #define I2C_SLAVE_ADDRESS_DBG		I2C_SLAVE_ADDRESS
 #define I2C_SLAVE_DEFAULT_INTR_MASK	0x48FF
 
